main.rb

```diff
@@ -60,9 +59,73 @@ def correct_name
   end
 end
 
-person = Person.new(22, 1, name: 'maximilianus')
+person = Person.new(age: 22, name: 'maximilianus', parent_permission: true)
 person.correct_name
 capitalized_person = CapitalizeDecorator.new(person)
 puts capitalized_person.correct_name
 capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
 puts capitalized_trimmed_person.correct_name
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
```

Comment:
- [ ] This suggestion from the previous review has not been attended to :point_down: 

```
class Student < Person
  attr_reader :classroom
  .
  .

  def classroom(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```
Kindly attend to it.